### ISSUE TITLE
Quest farm at A Girl Named Mika Episode 2 prematurely enters Party Selection mode

### ISSUE THREAD

**Issue by ark-royal-91** [steve1316/granblue-automation-pyautogui#121]
Per title, the Quest farm for Converging Rays at A Girl Named Mika Episode 2 progresses to the Quest window, then simply sits there and skips ahead to summon selection without selecting chapter, quest, or episode.
 
[log 2022-03-20 15-48-27.txt](https://github.com/steve1316/granblue-automation-pyautogui/files/8312350/log.2022-03-20.15-48-27.txt)

**Comment by colpocleisis** [comment]
there also seems to be a bug for locating the go button when moving to the island 
```
0:00:40 [ERROR] Bot encountered exception in Farming Mode: 
Traceback (most recent call last):
  File "C:\Users\xinyu\Desktop\gbf\backend\bot\game.py", line 871, in start_farming_mode
    Quest.start(first_run)
  File "C:\Users\xinyu\Desktop\gbf\backend\bot\game_modes\quest.py", line 502, in start
    Quest._navigate()
  File "C:\Users\xinyu\Desktop\gbf\backend\bot\game_modes\quest.py", line 475, in _navigate
    Quest._navigate_to_phantagrande_skydom_island(Settings.map_name, current_island)
  File "C:\Users\xinyu\Desktop\gbf\backend\bot\game_modes\quest.py", line 143, in _navigate_to_phantagrande_skydom_island
    raise QuestException(f"Failed to enter {map_name} as the Go button is missing.")
bot.game_modes.quest.QuestException: Failed to enter New Utopia as the Go button is missing.

**Comment by steve1316** [comment]
> there also seems to be a bug for locating the go button when moving to the island
> 
> ```
> 0:00:40 [ERROR] Bot encountered exception in Farming Mode: 
> Traceback (most recent call last):
>   File "C:\Users\pc\Desktop\gbf\backend\bot\game.py", line 871, in start_farming_mode
>     Quest.start(first_run)
>   File "C:\Users\pc\Desktop\gbf\backend\bot\game_modes\quest.py", line 502, in start
>     Quest._navigate()
>   File "C:\Users\pc\Desktop\gbf\backend\bot\game_modes\quest.py", line 475, in _navigate
>     Quest._navigate_to_phantagrande_skydom_island(Settings.map_name, current_island)
>   File "C:\Users\pc\Desktop\gbf\backend\bot\game_modes\quest.py", line 143, in _navigate_to_phantagrande_skydom_island
>     raise QuestException(f"Failed to enter {map_name} as the Go button is missing.")
> bot.game_modes.quest.QuestException: Failed to enter New Utopia as the Go button is missing.
> ```

Do you remember what island you were on before the bot attempted to move to The Edgelands?

**Comment by colpocleisis** [comment]
> Do you remember what island you were on before the bot attempted to move to The Edgelands?

Zinkenstill, and the issue happen when I try to run house of happiness

https://imgur.com/a/ogWZQjJ

```
0:00:02 [QUEST] Beginning process to navigate to the mission: House of Happiness...
0:00:03 [INFO] Bot is at the Home screen.
0:00:05 [WARNING] Failed to find the CLOSE button.
0:00:05 [WARNING] Failed to confirm the bot's location at MAP_NEW_UTOPIA.
0:00:05 [QUEST] Bot is currently not on the correct island.
0:00:06 [WARNING] Failed to confirm the bot's location at MAP_ZINKENSTILL.
0:00:06 [WARNING] Failed to confirm the bot's location at MAP_PORT_BREEZE_ARCHIPELAGO.
0:00:06 [WARNING] Failed to confirm the bot's location at MAP_VALTZ_DUCHY.
0:00:07 [WARNING] Failed to confirm the bot's location at MAP_AUGUSTE_ISLES.
0:00:07 [QUEST] Bot's current location is at Lumacie Archipelago. Now moving to New Utopia...
0:00:14 [WARNING] Failed to confirm the bot's location at YOU_RETREATED_FROM_THE_RAID_BATTLE.

0:00:36 [QUEST] Beginning process to navigate to the island inside the Phantagrande Skydom: New Utopia...
0:00:38 [WARNING] Failed to find the GO button.

0:00:38 [ERROR] Bot encountered exception in Farming Mode: 
Traceback (most recent call last):
  File "C:\Users\pc\Desktop\gbf\backend\bot\game.py", line 871, in start_farming_mode
    Quest.start(first_run)
  File "C:\Users\pc\Desktop\gbf\backend\bot\game_modes\quest.py", line 502, in start
    Quest._navigate()
  File "C:\Users\pc\Desktop\gbf\backend\bot\game_modes\quest.py", line 475, in _navigate
    Quest._navigate_to_phantagrande_skydom_island(Settings.map_name, current_island)
  File "C:\Users\pc\Desktop\gbf\backend\bot\game_modes\quest.py", line 143, in _navigate_to_phantagrande_skydom_island
    raise QuestException(f"Failed to enter {map_name} as the Go button is missing.")
bot.game_modes.quest.QuestException: Failed to enter New Utopia as the Go button is missing.

**Comment by steve1316** [comment]
Fixed in v1.5.2
